Assets/SEE/UI/Window/CodeWindow/CodeWindowContextMenu.cs

```diff
@@ -0,0 +1,310 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Cysharp.Threading.Tasks;
+using Cysharp.Threading.Tasks.Linq;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models;
+using SEE.Controls;
+using SEE.Controls.Actions;
+using SEE.Tools.LSP;
+using SEE.UI.Menu;
+using SEE.UI.Notification;
+using SEE.UI.PopupMenu;
+using SEE.Utils;
+using UnityEngine;
+using UnityEngine.Assertions;
+using Range = SEE.DataModel.DG.Range;
+
+namespace SEE.UI.Window.CodeWindow
+{
+    public partial class CodeWindow
```

Comment:
It would make sense to describe what this part implements analogously to `Assets/SEE/UI/Window/CodeWindow/CodeWindowInput.cs`.

Reply:
Fixed.